server/server.go

```diff
@@ -76,23 +144,40 @@ func setupStaticFiles(directory string, path string) {
 	http.Handle(path, http.StripPrefix(path, fileServer))
 }
 
-func setupRoutes() {
+func setupRoutes() *pool.Pool {
```

Comment:
It might be cleaner to pass in an already created pool here, rather than create a new pool and return it. You could move the creation to `StartLocalServer`

Reply:
Makes sense